src/callbacks_step/analysis_surface_2d.jl

```diff
@@ -0,0 +1,148 @@
+    @unpack psi, rhoinf, uinf, linf = lift_coefficient.force_state
+    n = dot(normal_direction, psi) / norm(normal_direction)
+    return p * n / (0.5 * rhoinf * uinf^2 * linf)
+end
+
+function (drag_coefficient::DragCoefficient)(u, normal_direction, equations)
+    p = pressure(u, equations)
+    @unpack psi, rhoinf, uinf, linf = drag_coefficient.force_state
+    n = dot(normal_direction, psi) / norm(normal_direction)
+    return p * n / (0.5 * rhoinf * uinf^2 * linf)
+end
+
+function analyze(surface_variable::AnalysisSurfaceIntegral, du, u, t,
+                 mesh::P4estMesh{2},
+                 equations, dg::DGSEM, cache)
+    @unpack boundaries = cache
+    @unpack surface_flux_values, node_coordinates, contravariant_vectors = cache.elements
+    @unpack weights = dg.basis
+    @unpack semi, indices, variable = surface_variable
+
+    surface_integral = zero(eltype(u))
+    index_range = eachnode(dg)
+    for local_index in eachindex(indices)
+        # Use the local index to get the global boundary index from the pre-sorted list
+        boundary = indices[local_index]
+
+        # Get information on the adjacent element, compute the surface fluxes,
+        # and store them
+        element = boundaries.neighbor_ids[boundary]
+        node_indices = boundaries.node_indices[boundary]
+        direction = indices2direction(node_indices)
+
+        i_node_start, i_node_step = index_to_start_step_2d(node_indices[1], index_range)
+        j_node_start, j_node_step = index_to_start_step_2d(node_indices[2], index_range)
+
+        i_node = i_node_start
+        j_node = j_node_start
+        for node_index in eachnode(dg)
+            u_node = Trixi.get_node_vars(cache.boundaries.u, equations, dg, node_index,
+                                         boundary)
+            normal_direction = get_normal_direction(direction, contravariant_vectors,
+                                                    i_node, j_node,
+                                                    element)
+
+            # L2 norm of normal direction is the surface element
+            dS = weights[node_index] * norm(normal_direction)
+            surface_integral += variable(u_node, normal_direction, equations) * dS
+
+            i_node += i_node_step
+            j_node += j_node_step
+        end
+    end
+    return surface_integral
+end
+
+function pretty_form_ascii(::AnalysisSurfaceIntegral{<:Any,
+                                                     <:LiftCoefficient{<:Any}})
+    "CL"
+end
+function pretty_form_utf(::AnalysisSurfaceIntegral{<:Any,
+                                                   <:LiftCoefficient{<:Any}})
+    "CL"
+end
+function pretty_form_ascii(::AnalysisSurfaceIntegral{<:Any,
+                                                     <:DragCoefficient{<:Any}})
+    "CD"
+end
+function pretty_form_utf(::AnalysisSurfaceIntegral{<:Any,
+                                                   <:DragCoefficient{<:Any}})
+    "CD"
+end
```

Comment:
As we now have in principle the possibility to compute coefficients for multiple boundaries, this should probably reflect that. Maybe we can pass in a string for the force to be printed.

Reply:
Could you give an idea of how you wish to indicate multiple boundaries? Do you suggest printing the symbols whose forces are being computed?

Reply:
Hm not sure, as of now I think it is not really common to have different stage callbacks of different kind.